IDEAS/Buildings/Components/Shading/HorizontalFins.mo

```diff
@@ -30,6 +31,8 @@ protected
 
   Modelica.Blocks.Interfaces.RealInput beta_internal
     "Internal variable for inclination angle";
+  Modelica.Blocks.Interfaces.RealInput Ctrl_to_beta_internal
+    "Internal variable to linearly map the Ctrl input [0,1] onto the fin inclination angle [beta_min=0,beta_max]";
```

Comment:
I'm not sure if we need to introduce this new input, one of these should suffice I think.

Reply:
There is an implementation possible where only two of the three internal inputs are needed (`beta_internal`, `Ctrl_to_beta_internal`, `disp_internal`), but that would make the model less readible for outsiders, so let's keep this as is.